### DISCUSSION TITLE
Why I can't vote?

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#767] [Q&A] [answered]
Suggest answers below or vote for an answer.

**Comment by metrox-eth** [comment, marked as answer]
To be able to vote you need voting power and there are several possible reasons why your voting power may be zero. Some of the most common are:

1. You didn't have the tokens in your wallet before the proposal "snapshot". You can find the proposal snapshot above your voting power when you are casting a vote. <img width="796" alt="image" src="https://user-images.githubusercontent.com/51686767/197967366-57805758-7071-4c5a-bcde-d79537876c8b.png">

2. You haven't enabled governance for your tokens. Some spaces require you to delegate/self-delegate the tokens to enable governance. This can usually be done through the governance website of the organization. Popular space that require governance enabled tokens: 
  - Optimism - https://app.optimism.io/delegates
  - Uniswap - https://app.uniswap.org/#/vote
  - ENS - https://claim.ens.domains/delegate-ranking
  - Gitcoin - http://decentralized.quadraticlands.com/

3. Your tokens are on a chain that isn't configured or they are deposited into a lending protocol like Aave.

4. Each project on Snapshot has it's own governance flow and setup, documentation is a great way to learn about their individual governance process.

**Reply by csggene3** [reply]
There is also something else going on. I clearly had tokens at the snapshot for the DAO/proposal I'm looking at but hadn't "joined" on snapshot yet. There could be other factors as well, like the network you're wallet was connected to when joining the DAO's snapshot, or anything else I'm not aware of. But these qualifiers should be broadcasted front and center when good governance and the concept of inclusivity are central to this effort.

**Reply by metrox-eth** [reply]
You can definitely vote without having joined a space.

**Reply by AL0YSI0US** [reply]
I'm in the same boat...

I can't vote, have had tokens for a while and just cant vote.  I got tokens somewhere in my beginning Journey at the Token Engineering Commons many months back. I could have said this a while ago but I guess I wasnt that bothered by my inability to support governance proposals.

**Reply by metrox-eth** [reply]
> I'm in the same boat...
> 
> I can't vote, have had tokens for a while and just cant vote. I got tokens somewhere in my beginning Journey at the Token Engineering Commons many months back. I could have said this a while ago but I guess I wasnt that bothered by my inability to support governance proposals.

Please kindly verify that the token you own and the token used in the strategy contract of the space you want to vote is the same. Could be different causes:
- Slightly different token  (Staked, governance token, ...) 
- Your token has been delegated or require delegation (ENS, UNI, ...)
Can you ask the project what is their governance requirements ?

**Reply by CalmanChen** [reply]
How many $PEOPLE should I hodl to be qualified to be able to vote?Thanks.

**Reply by mk4810** [reply]
How does paop prove that he has the right to vote?

**Reply by L4CB** [reply]
不能投票，已经有一段时间了，只是不能投票。几个月前，我在 Token Engineering Commons 开始旅程的某个地方获得了代币。我之前可以这么说，但我想我并没有因为无法支持治理提案而感到困扰。

**Reply by HadeelY** [reply]
i can't vote

**Reply by 0xerison** [reply]
I bought a fraction of Bobu #40 to be able to vote, and I can't, does anyone know what the restriction is? Why is the fraction for sale if you can't vote? Thankss

**Reply by 0xerison** [reply]
Ow I understand now, sorry for the ignorance I believe I missed a part of the answer for newbies, we don't know the meaning of the blocks.
If the proposal was created before you acquired the token, you cannot vote, only on future proposals after acquisition.

**Reply by toubazi** [reply]
i have alpaca in my wallet ,but i have no right to vote,why? did it have a balance limit ?

**Reply by SaveSnowflake** [reply]
I moved my ape yesterday, now voting gives me zero ape. Why? I am connected with the wallet where ape is. Thank you.

**Reply by metrox-eth** [reply]
Check the creation date (snapshot) of the proposal you want cast a vote on. If the proposal has been created before you've received the NFT in your wallet, you won't have voting power, you will be allowed to vote on the next proposal (if you hold that token).

**Reply by polexiy** [reply]
I have the same problem too. I can`t vote in 1 wallet which has the same characteristics of holding etc as the wallet where I can vote. Arbitrum

**Reply by metrox-eth** [reply]
> I have the same problem too. I can`t vote in 1 wallet which has the same characteristics of holding etc as the wallet where I can vote. Arbitrum

Would you mind come on Discord https://discord.snapshot.org/ or Telegram https://t.me/snapshotlabs so we can troubleshot this.
Githhub discussions is more a FAQ than support. Not very convenient.

**Reply by amirhabibzadeh** [reply]
I think you can show user what is the problem. Example message : "You've been get your tokens after the proposal creation, so you can't vote on this proposal."

**Reply by metrox-eth** [reply]
> I think you can show user what is the problem. Example message : "You've been get your tokens after the proposal creation, so you can't vote on this proposal."

You have an idea on how to implement this ?

**Reply by Milliondollaroad** [reply]
i can't vote too. i have the same issue

**Reply by Milliondollaroad** [reply]
something wrong . i got my gnosis 3 days ago but still can't vote !

**Reply by Hamid40771** [reply]
Done Together we can think big and get better results. This is right and good

**Reply by Vevivo** [reply]
Instant Impermanent Loss Protection is offered by the protocol; users no longer need to wait 100 days for 100% IL protection

**Reply by Fazar87** [reply]
I'm in the same boat...

I can't vote, have had tokens for a while and just cant vote. I got tokens somewhere in my beginning Journey at the Token Engineering Commons many months back. I could have said this a while ago but I guess I wasnt that bothered by my inability to support governance proposals.

**Reply by vanhai97** [reply]
I used to trade before snapshot date. but I can't vote. what's wrong ??//

**Reply by metrox-eth** [reply]
For those who can't vote please send a link to the proposal you are trying to vote on. I'll help.

**Reply by Taurusbsc** [reply]
https://snapshot.org/#/arbitrum-odyssey.eth/proposal/0x44aba87414d2d7ce88218b676d9938338d7866a245f48a7829e805a99bcda6a2

**Reply by metrox-eth** [reply]
To vote on this proposal you need minimum 0.0002 ETH on Arbitrum chain before the date of the creation of the proposal.

**Reply by jetskie43** [reply]
I cant vote

**Reply by mjraj9612** [reply]
Iunable to vote 😣 voting power 0 showing the Am also same bot
I have L2dao tokens, ETH 
& sheel PROTOCAL NFT also have I have done lot of transection in uniswap Arbitram
Please recheck once & is clear the issue

**Reply by Humayun49** [reply]
Why i am not vote ? Still my wallet hold 0.0002 AETH - 0xF511470999e3F45550b886a3b483cDE0AD1B7201

It's not fair to me many time try to vote

**Reply by AdoNunes** [reply]
Check that the correct wallet is connected!

**Reply by MartinsKingjr** [reply]
> You can definitely vote without having joined a space.

please how can I do that because I've been trying since yesterday.. please help 
me out . thanks in anticipation

**Reply by ZonaAirdrop** [reply]
Good projects

**Reply by npmita** [reply]
Nice platform

**Reply by boyar9** [reply]
nice project

**Reply by ASISBusiness** [reply]
> To be able to vote you need voting power, your voting power is calculated at the "snapshot" of the proposal. The snapshot is the block number where we read the voting power from. If you didn't have the tokens at this block number, you will not have any voting power and won't be able to vote.

**Reply by GastonGuan** [reply]
For BitDao, you have to delegate your vote power to yourself or another address first. You can follow this guide:  https://docs.bitdao.io/litepaper-1/how-to-delegate

**Reply by edipcivi** [reply]
hello, I just sent ape tokens to my wallet, when can I be eligible to vote?

**Reply by metrox-eth** [reply]
You'll be able to vote on the next proposal is you hold the token.

**Reply by rose5620** [reply]
I have Dexguru Nft ,but I can't vote

**Reply by metrox-eth** [reply]
If you got the token after the snapshot of the proposal, you will be able to vote on the next proposal, given that you still hold that token.

**Reply by ced185crypto** [reply]
@metrox-eth this is not true, i bought and staked my GNO via stake.wise, 1 month ago with this wallet 0x54cC5Ed3593C5a54E053dF1d09a9D5aB1b0dd626  haven't been able to vote to any proposal since

**Reply by metrox-eth** [reply]
Does Gnosis's Snapshot space have a strategy that allow voting power from staked token ? You can check with them for the voting requirements.

**Reply by SirQuincy** [reply]
how do i get voting power then

**Reply by apzadmin** [reply]
I can vote

**Reply by Etgggggg** [reply]
I can vota

**Reply by jununmuni** [reply]
I can't vote

**Reply by Palmbobo** [reply]
I can't vote

**Reply by Pain2691** [reply]
what is recquirements?to gain voting power?

**Reply by apzadmin** [reply]
I can't vote

**Reply by Atike123** [reply]
I can't vote

**Reply by Atike123** [reply]
how to get voting power

**Reply by talabit2** [reply]
I can't vote

**Reply by Kornelrozsa** [reply]
Same here got the ZAPPER NFT & use the platform all the times still cant vote.

**Reply by rahulkahar0** [reply]
why i m not able to vote

**Reply by Vitamin82** [reply]
hi

**Reply by MdDilnawazHus13** [reply]
I can't vote why

**Reply by Smartboy06** [reply]
i cant vote why

**Reply by Smartboy06** [reply]
i cani vote why

**Reply by zombielr** [reply]
我不能投票为什么

**Reply by sawyeryts** [reply]
i can not vote

**Reply by weitingGao** [reply]
why i can not vote

**Reply by kishore594** [reply]
I have POAP NFTs in GENO chain. Showing zero voting power. How to become eligible for POAP.

**Reply by X-MG** [reply]
我怎么不能投票？

**Reply by X-MG** [reply]
why i can not vote

**Reply by yoshimo100** [reply]
i can't vote...

**Reply by iliyaaazad** [reply]
i can't vote...

**Reply by lebedeviy** [reply]
Yes

**Reply by Humayun49** [reply]
Can check plz hold nft but can't vote why ?

0xF511470999e3F45550b886a3b483cDE0AD1B7201

**Reply by Mr-kay-cloud2** [reply]
How do we get tokens to vote?

**Reply by 846200** [reply]
https://docs.google.com/document/d/1Vikc74CeRrkKl5MOuiGmohWfSq2ft9HnK1uoGXBZIlM/edit

**Reply by 0X-UglyFrog** [reply]
yes

**Reply by ikun2020** [reply]
Yes

**Reply by Timmy6666** [reply]
Just ask why i can't vote.

**Reply by csf1349** [reply]
yes

**Reply by puggyFF** [reply]
same. I have some coin in my wallet, but in snapshot is shows i have 0 so i cant vote.

**Reply by Zhao4Nicholas** [reply]
How many tokens do I need to keep in order to maintain my voting governance?

**Reply by dawico121** [reply]
Nice!

**Reply by 0x8a54** [reply]
oo

**Reply by APrimaveraV** [reply]
https://github.com/metrox-eth

Hi from Dominican Republic Sr.👋🏼🇩🇴

 Please can some busy tell me what I can do if I not have recourses for vote but at same time we really wanna continue learning abouth.?

**Reply by go8282** [reply]
🌈your project is very good, the whitepaper is also very clear, I hope your project can be successful and successful in the future, I also hope the community can grow even bigger than now

**Reply by NIKAN1997** [reply]
I can't vote, have had tokens for a while and just cant vote. I got tokens somewhere in my beginning Journey at the Token Engineering Commons many months back

**Reply by Jostos1** [reply]
Im new to these DAO voting please. Which way to go?

**Reply by licunbao** [reply]
我获得了op的空投        0xEAa8Cd032240623Abb37AfD6D367fa1deEb05799

**Reply by 19812021** [reply]
i have 132 op tokens in rubicon exchenge but not able to vote.
what is the solution.

**Reply by wangxingruo** [reply]
I can vota too

**Comment by NSR2519s** [comment]
So i have $fbeets in my wallet and am still unable to vote for a snapshot that is due on 20th feb. Any ideas why? Need HELP!!!

**Reply by metrox-eth** [reply]
What's the proposal ? if it's on Fantom chain they are having issues with all nodes atm.

**Reply by NSR2519s** [reply]
Thanks for the prompt response.
Yeah its on BeethovenX. FTM-Credit_Sonata pool. 
Any idea how long till the node issues are resolved?

**Reply by apzadmin** [reply]
Thanks for the prompt response.

**Comment by metrox-eth** [comment]
I've pinged Chainstack who pinged them :-)

**Reply by NSR2519s** [reply]
Thanks a ton.

**Reply by metrox-eth** [reply]
Looks like Fantom node is back to full archive state, can you confirm ?

**Reply by mungeri12** [reply]
I have the same issue. looks like this hasn't been resolved. ANY help would be much appreciated. Thanks.

**Reply by apzadmin** [reply]
Thanks

**Comment by Jackywwl** [comment]
I finished the task of rabbithole in project, i.e swap token and deposit asset on uniswap for etherium network yesterday 23rd Feb 2022, but didnot get the debit score till now. could you pls help solve it?

**Comment by Scottmiller666** [comment]
Why can’t I vote

**Reply by WUXINYIMiffy** [reply]
To be able to vote you need voting power, your voting power is calculated at the "snapshot" of the proposal. The snapshot is the block number where we read the voting power from. If you didn't have the tokens at this block number, you will not have any voting power and won't be able to vote.

**Reply by Usajim** [reply]
Whats snapshot?
Whe was it taken?

**Comment by Scottmiller666** [comment]
Please I want to be among the qualify people  to vote

**Comment by Ojeyin** [comment]
how can i qualify for voting

**Comment by AndrewPech** [comment]
Путин будет в аду со всеми кто поддержываете старого марозматика и розвязал войну против моей страны.

**Comment by likun9** [comment]
I want to get the right to vote

**Reply by 19812021** [reply]
i have 132 op tokens staked on rubicon dex but not able to vote.
how could i be able to do this pls

**Comment by HadeelY** [comment]
i can't vote

**Comment by Middonight** [comment]
Where do we need to hold our Apecoins in order to be eligible for voting? 
I currently have them on an exchange. Do I need to transfer them to my Metamask? 

Can you please clarify on this?

**Reply by metrox-eth** [reply]
Yes the tokens need to be on the address you vote with before the date of creation of the proposal. (Snapshot) Transfer them on your MetaMask address and you will be able to vote on future proposals.

**Comment by CalmanChen** [comment]
OK，thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月24日(星期四) 下午2:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [snapshot-labs/snapshot] Why I can&#39;t vote? (Discussion #767)





 
Yes the tokens need to be on the address you vote with before the date of creation of the proposal. (Snapshot) Transfer them on your MetaMask address and you will be able to vote on future proposals.

**Reply by asaam7373** [reply]
احسنت

**Comment by SaveSnowflake** [comment]
Staking should not be longer then 1 year. Better 6 month. Or options 3 month , 6, 1 year, 2, 3

**Reply by metrox-eth** [reply]
Is this comment related to Snapshot.org ?

**Comment by pianokw** [comment]
I staked all ilv, so i cant vote???

**Reply by WUXINYIMiffy** [reply]
Basically it depends on strategy, if the strategy is for staked token, then you can vote.

**Comment by BVK0515** [comment]
So i have $fbeets in my wallet and am still unable to vote for a snapshot that is due on 20th feb. Any ideas why? Need HELP!!!

**Comment by TaBachtl** [comment]
Nice

**Comment by mochitoqS** [comment]
I have token staked in stakewise.io but can't vote : someone knows why ?

**Comment by md113366** [comment]
good job

**Reply by pwwwwww** [reply]
very good

**Comment by Takana0624** [comment]
good job

**Comment by joviachen** [comment]
how can i get vote apprication?

**Comment by naga186** [comment]
Hi! For Uniswap space only delegated UNI may be used to vote on proposals. You can delegate to yourself or another address here: https://app.uniswap.org/#/vote

**Comment by Humayun49** [comment]
This is my corret wallet address -
0xF511470999e3F45550b886a3b483cDE0AD1B7201
Still now no voting power can you check plz sir ,

**Comment by Doniprasetyo2011** [comment]
Great

**Comment by Raisa445** [comment]
I can vote a project

**Comment by vuongquy8888** [comment]
treasure dao

**Comment by jetskie43** [comment]
How to vote

**Comment by futre1** [comment]
how can i qualify for voting?

**Comment by pderrico49** [comment]
Can't vote
0xB373CD7F2c81bbc59D909A222B120C2689Ac00C2
pamelanotpam.eth (same account)
I have 3 NFTs

**Comment by saman616** [comment]
I want to get the right to vote

**Comment by alexkaaa** [comment]
Why can't I vote if I have a PreZKP token?

**Comment by TaraThu1** [comment]
Hi Ad, Why i can not vote for Amara finace?

**Reply by apzadmin** [reply]
How should we vote?
Please help

**Comment by Scottmiller666** [comment]
Yes getting the vote power is the right thing to do first

**Comment by Abuchtela** [comment]
> Suggest answers below or vote for an answer.

**Comment by Biscuitzy** [comment]
无法投票，如何解决呢

**Comment by blocksurance-dao** [comment]
怎么获得投票权 需要持有哪个代币吗

**Reply by iskander33** [reply]
😡

**Comment by AiontheWise** [comment]
Hello,
I have bought one Lille regular card by credit card and did not transfer it to a wallet because I did not need to. Today I want to vote but I can't as I have nothing on my wallet and the snapshot is already done. Why people who bought by credit card and keep it in the website can't vote ? Or if I can please tell me how. Thanks

**Reply by metrox-eth** [reply]
Your wallet address is what we use to identify you, you don't really have a wallet in an exchange. On Snapshot proposals contain a block number that is the closest to it's creation, to avoid ppl transferring token and influence the outcome.

**Reply by AiontheWise** [reply]
Ok but in that case my metamask adress that I am using to connect with Wincity should grant me 1 NFT no ? When I connect my metamask wallet to the vote site I don't have any right to vote

**Comment by frankiefab100** [comment]
I can't vote on POAP.xyz proposal. How do I get voting power?

**Comment by MarkZa007** [comment]
I have 0.05 stETH but I cannot vote

**Comment by APrimaveraV** [comment]
Hi Sr 👋🏼🇩🇴

**Comment by iskander33** [comment]
Pues yo ni se como hacerme una transacción en bbsscc comore una tarjeta PayPal xon un volto de 500€ y no puedo trasferirmelos alguien puede ayudarme tengo el número de contrato y el Has

**Comment by APrimaveraV** [comment]
hi again guys, can some budy contact me via direc for help to understan in
correct ways how i need to continue learning about whit all you. ?
